### ISSUE TITLE
Could I have your retrained resnet50 on imagenet?

### ISSUE THREAD

**Issue by ginger0106** [guoxiaolu/model_compression#1]
Thanks for your contribution. Could I have your retrained resnet50 on imagenet? I do not have enough resources to retrain it. Could you help me? Thanks a lot.

**Comment by guoxiaolu** [comment]
sorry, currently, I don't have the retrained model, I have deleted it. It's not that difficult to retrain it if you have imagenet dataset and gpu~

**Comment by ginger0106** [comment]
hi, I am trying to retrain the pruned resnet50, using 45000 val pics of imagenet. The std is set to 1, but the inference accuracy stays stable at 55% for top5, after 100 epochs... How many epochs and how many pics did you use? Thank you very much.  By the way,  could I have your email or any other ID?

**Comment by guoxiaolu** [comment]
Sorry for late reply. When I retrain it, I cannot get the official top1 error. So I just compare the initial weights and pruned weights, these two weights achieved almost the same top5 error. the initial and the pruned top5 error achieved about 65%. The more of the val pics number, the better. I used about 200k val random pics. You may try this when evaluating, https://github.com/keras-team/keras/issues/8672. But I don't have time to test whether the solution is correct or not. The first thing to do is to evaluate the imagenet error.
This solution finally is not used in product as it is time consuming when retraining and the hidden evaluation problem. Thank you, may it helps you~

**Comment by ginger0106** [comment]
Thanks for the help and sorry for the late reply, I used the whole Imagenet and updated my codes, the top5 is up to 88% after 6 epochs fine-tune. Besides, have you ever tried vgg? I pruned vgg16, but the pruned model could not convergence in retrain process.

**Comment by guoxiaolu** [comment]
To my knowledge, vgg is slower to converge, and from keras.io you can find the keras model performance, it is lower than the paper. I don't test vgg on imagenet.
88% is pruning result or just no pruning model result?

**Comment by ginger0106** [comment]
88% is the pruned model result for std=1.0, I tested std=0.8,0.6 and 0.4, the results are all above 80%.   For the retrain of pruned VGG16, the acc is around 1e-4 after 1.5 days...and the loss would be descended...

**Comment by guoxiaolu** [comment]
Thank you for your result, the hidden problem in Readme is solved. Thank you very much.
As the classical VGG-like model doesn't have the better performance as inception-like and resnet-like model in real world, I don't test it on imagenet. I think the fully-connected layer also needs to be pruned just as the job at the beginning of this field, more parameters can be pruned in VGG.

**Comment by ginger0106** [comment]
Thank you for your help, I reduced the learning rate so that the VGG is tend to convergence now, thanks.
